great-idea-website/index.html

```diff
@@ -15,49 +16,106 @@
 </head>
 
   <body>
+<header class="page_header">
```

Comment:
I would have wrapped the whole page in a `container` class and set it to the width of the middle image, which was `876px`.  You would have had better time of laying out the page according to the design file.